x-pack/plugins/secops/public/components/timeline/body/actions/index.tsx

```diff
@@ -30,34 +30,15 @@ interface Props {
   updateNote: UpdateNote;
 }
 
-const ActionsContainer = styled.div`
-  align-items: center;
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
+const ActionsContainer = styled(EuiFlexGroup)`
   min-width: ${ACTIONS_COLUMN_WIDTH}px;
   overflow: hidden;
-  width: ${ACTIONS_COLUMN_WIDTH}px;
-`;
-
-const ActionsRows = styled.div`
-  align-items: center;
-  display: flex;
-  flex-direction: column;
-`;
-
-const ActionsRow = styled.div`
-  align-items: center;
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-  min-width: ${ACTIONS_COLUMN_WIDTH}px;
-  padding: 2px 8px 0 0;
+  padding: 2px 10px 0 0;
```

Comment:
Mentioning that I am seeing a mix of `rem` vs `px`

```
padding: 1rem 1rem 0rem 1rem;
```

Don't know if we need to have guidelines around this but pointing it out.

Reply:
Let's have a team discussion around this, because yes, both are used throughout.